tensorflow/compiler/mlir/tensorflow/ir/tf_ops_n_z.cc

```diff
@@ -1801,10 +1797,7 @@ static LogicalResult Verify(TransposeOp op) {
       const int64_t y_dim = y_type.getDimSize(y_idx);
       const int64_t x_idx = e.value().getSExtValue();
       const int64_t x_dim = x_type.getDimSize(x_idx);
-      if (y_dim == ShapedType::kDynamicSize || x_dim == ShapedType::kDynamicSize) {
-        continue;
-      }
-      if (y_dim != x_dim) {
+      if (y_dim != ShapedType::kDynamicSize && x_dim != ShapedType::kDynamicSize && y_dim != x_dim) {
```

Comment:
This seems to be exceeding 80 char limit, break it into two lines.
https://google.github.io/styleguide/cppguide.html#Line_Length

Reply:
Sorry that I commit in a rush, and forget to run clang-format.